app/posts/register-trainee-teachers/2020-09-07-register-teacher-trainees-alpha-prototype.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Register Trainee Teachers - alpha prototype 
+description: A look at the alpha prototype and beginning to think about the next steps.
+date: 2020-09-07
+related:
+  items:
+  - text: Alpha assessment
+    href: https://docs.google.com/presentation/d/1Ws9gmeHZTjmc2NRZlSCcVEfReEmyi9KB1_BLyyJv-ZE/edit?usp=sharing
+  - text: Alpha research overview
+    href: https://dfedigital.atlassian.net/wiki/spaces/TTDT/pages/1626931201/3.+User+Research+Rounds+Sprint+Work
```

Comment:
```suggestion
    href: https://dfedigital.atlassian.net/wiki/spaces/TTDT/pages/1626931201/
```

Which part of the URL is necessary (suspect the later part may change?)

Reply:
Either works. The cleaner version redirects to the lonnger version.